controllers/api/authenticate.js

```diff
@@ -294,21 +294,21 @@ function check_create_token_request(req) {
 }
 
 
-// Function to check password method parameter for identity
+//  Function to check password method parameter for identity
 function search_identity(name, password) {
 
 	return new Promise(function(resolve, reject) {
 
 		models.helpers.search_pep_or_user(name).then(function(identity) {
-
+			console.log("authenticate.js search_pep_or_user(name).then(function(identity) identity: ", identity);
```

Comment:
This should be a `debug` statement not a `console.log`

Reply:
remain from tests, will remove